hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenHandlers.java

```diff
@@ -99,7 +100,8 @@ public TokenHandlers(
             @NonNull final TokenGetInfoHandler tokenGetInfoHandler,
             @NonNull final TokenGetAccountNftInfosHandler tokenGetAccountNftInfosHandler,
             @NonNull final TokenGetNftInfoHandler tokenGetNftInfoHandler,
-            @NonNull final TokenGetNftInfosHandler tokenGetNftInfosHandler) {
+            @NonNull final TokenGetNftInfosHandler tokenGetNftInfosHandler,
+            TokenUpdateNftsHandler tokenUpdateNftsHandler) {
```

Comment:
Nit: `@NonNull final`

Reply:
Good catch! @thomas-swirlds-labs @Neeharika-Sompalli Please fix this in a follow-up PR.